packages/react/src/components/Forms/Input/index.spec.tsx

```diff
@@ -1,4 +1,4 @@
-import { render, screen } from "@testing-library/react";
+import { act, render, screen } from "@testing-library/react";
```

Comment:
User events by default are already wrapped with `act()`right ? ([I refer to this medium blog post](https://flyingsquirrel.medium.com/when-should-i-use-act-in-react-testing-library-d7dd22a3340e))
Plus, `act` warnings are silenced, should we ignore them ? They might inform us that some part of the tests haven't the right behavior